src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -577,11 +581,12 @@ def listFileBlockLocation(self, fileBlockName, dbsOnly = False, phedexNodes=Fals
                 msg += "%s\n" % str(ex)
                 raise Exception(msg)
 
-            if not blockInfo or len(blockInfo) != len(blockNames): #if we couldnt get data location from PhEDEx, try to look into origin site location from dbs
-                dbsOnly = True
-                blockNames = set(blockNames) - set(blockInfo) #get the blocks we did not find information in phedex
+            if blockInfo and len(blockInfo) == len(blockNames):
+                phedexOnly = True
```

Comment:
Are we talking about the same thing? I was referring to the question @mmascher made above (if it makes sense to have mixed locations from phedex and dbs). If @ericvaandering is in favor of testing it, then I first have to implement it. Should I? Or should we leave it for another issue.